ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-rhoam-addons__v4.13.yaml

```diff
@@ -0,0 +1,39 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: rhods-rhoam-tests
+  cron: 0 12 * * 6
```

Comment:
the job will be triggred when a new rhods or rhoam addon is released; plese update `cron` to run on a yearly basis.